doc/releases.rst

```diff
@@ -1,6 +1,29 @@
 Releases
 ========
 
+Version 0.9.0
+-------------
+
+Date: 2020-03-12
+
+This is a major release primarily for compatibility with the recent Bokeh 2.0 release. Additionally this release has a small number of features and bug fixes:
+
+Features:
+
+- Added a MultiChoice widget (`#1140 <https://github.com/holoviz/panel/pull/1140>`_)
+- Add FileDownload widget (`#915 <https://github.com/holoviz/panel/pull/915>`_, `#1146 <https://github.com/holoviz/panel/pull/1146>`_)
+- Add ability to define Slider format option (`#1142 <https://github.com/holoviz/panel/pull/1142>`_)
+- Expose `pn.state.cookies` and `pn.state.headers` (`#1143 <https://github.com/holoviz/panel/pull/1143>`_)
```

Comment:
Can you explain what the pn.state changes help achieve, e.g.  "to make it possible for apps to respect the current state" (or whatever)?